src/commands/wavy.js

```diff
@@ -1,9 +1,27 @@
+import axios from 'axios'
+import Discord from 'discord.js'
 import DiscordCommand from './command.js';
 
 export default class WavyCommand extends DiscordCommand {
-    async execute() {
-        const wavy = this.getEmojiByName('wavy');
-        const wavyLeft = this.getEmojiByName('wavyLeft');
-        this.message.channel.send(`${wavy} ${wavyLeft}`);
+  async execute() {
+    if (!this.member) {
+      this.send("You must be in a Roo Bot enabled Discord server to use this command");
+      return;
     }
+
+    const wavy = this.getEmojiByName('wavy');
+    this.message.react(wavy);
+
+    axios.get(
```

Comment:
probably could be more clear if await was used here.

```suggestion
    const response = await axios.get(
```